INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /admin/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 233, in _get_session
    return self._session_cache
AttributeError: 'SessionStore' object has no attribute '_session_cache'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "django_session" does not exist
LINE 1: ...ession_data", "django_session"."expire_date" FROM "django_se...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 251, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 222, in inner
    if not self.has_permission(request):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 196, in has_permission
    return request.user.is_active and request.user.is_staff
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/functional.py", line 240, in inner
    self._setup()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/functional.py", line 376, in _setup
    self._wrapped = self._setupfunc()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/middleware.py", line 23, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/middleware.py", line 11, in get_user
    request._cached_user = auth.get_user(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 174, in get_user
    user_id = _get_user_session_key(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 58, in _get_user_session_key
    return get_user_model()._meta.pk.to_python(request.session[SESSION_KEY])
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 65, in __getitem__
    return self._session[key]
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 238, in _get_session
    self._session_cache = self.load()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 43, in load
    s = self._get_session_from_db()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 32, in _get_session_from_db
    return self.model.objects.get(
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 425, in get
    num = len(clone)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 269, in __len__
    self._fetch_all()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 1308, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/sql/compiler.py", line 1156, in execute_sql
    cursor.execute(sql, params)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "django_session" does not exist
LINE 1: ...ession_data", "django_session"."expire_date" FROM "django_se...
                                                             ^

ERROR Internal Server Error: /admin/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 233, in _get_session
    return self._session_cache
AttributeError: 'SessionStore' object has no attribute '_session_cache'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "django_session" does not exist
LINE 1: ...ession_data", "django_session"."expire_date" FROM "django_se...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 251, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 222, in inner
    if not self.has_permission(request):
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/admin/sites.py", line 196, in has_permission
    return request.user.is_active and request.user.is_staff
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/functional.py", line 240, in inner
    self._setup()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/utils/functional.py", line 376, in _setup
    self._wrapped = self._setupfunc()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/middleware.py", line 23, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/middleware.py", line 11, in get_user
    request._cached_user = auth.get_user(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 174, in get_user
    user_id = _get_user_session_key(request)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 58, in _get_user_session_key
    return get_user_model()._meta.pk.to_python(request.session[SESSION_KEY])
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 65, in __getitem__
    return self._session[key]
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 238, in _get_session
    self._session_cache = self.load()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 43, in load
    s = self._get_session_from_db()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 32, in _get_session_from_db
    return self.model.objects.get(
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 425, in get
    num = len(clone)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 269, in __len__
    self._fetch_all()
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 1308, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/models/sql/compiler.py", line 1156, in execute_sql
    cursor.execute(sql, params)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/gustavomezzomo/.virtualenvs/projeto_mercado/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "django_session" does not exist
LINE 1: ...ession_data", "django_session"."expire_date" FROM "django_se...
                                                             ^

ERROR "GET /admin/ HTTP/1.0" 500 145
ERROR "GET /admin/ HTTP/1.0" 500 145
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /admin/ HTTP/1.0" 302 0
INFO "GET /admin/ HTTP/1.0" 302 0
INFO "GET /admin/login/?next=/admin/ HTTP/1.0" 200 2194
INFO "GET /admin/login/?next=/admin/ HTTP/1.0" 200 2194
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/login.css
WARNING Not Found: /static/admin/css/login.css
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING "GET /static/admin/css/login.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/login.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /admin/ HTTP/1.0" 200 6593
INFO "GET /admin/ HTTP/1.0" 200 6593
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/base.css
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING Not Found: /static/admin/css/dashboard.css
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/dashboard.css
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/css/dashboard.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/css/dashboard.css HTTP/1.0" 404 179
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /api/image
WARNING Not Found: /api/image
WARNING "GET /api/image HTTP/1.0" 404 179
WARNING "GET /api/image HTTP/1.0" 404 179
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /api/image
WARNING Not Found: /api/image
WARNING "GET /api/image HTTP/1.0" 404 179
WARNING "GET /api/image HTTP/1.0" 404 179
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /api/image
WARNING Not Found: /api/image
WARNING "GET /api/image HTTP/1.0" 404 179
WARNING "GET /api/image HTTP/1.0" 404 179
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /api/image
WARNING Not Found: /api/image
WARNING "GET /api/image HTTP/1.0" 404 179
WARNING "GET /api/image HTTP/1.0" 404 179
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR "GET /api/image HTTP/1.0" 500 145
ERROR "GET /api/image HTTP/1.0" 500 145
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR "GET /api/image HTTP/1.0" 500 145
ERROR "GET /api/image HTTP/1.0" 500 145
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR Internal Server Error: /api/image
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 62, in image
    return JsonResponse(post.to_json_dict())
  File "/home/gustavomezzomo/projeto_mercado/blog/models.py", line 18, in to_json_dict
    'image': self.image.image
AttributeError: 'ImageFieldFile' object has no attribute 'image'
ERROR "GET /api/image HTTP/1.0" 500 145
ERROR "GET /api/image HTTP/1.0" 500 145
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 16, in post_list
    return render(request, 'blog/post_list.html', {'posts': posts})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: blog/post_list.html
ERROR Internal Server Error: /
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 16, in post_list
    return render(request, 'blog/post_list.html', {'posts': posts})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: blog/post_list.html
ERROR "GET / HTTP/1.1" 500 145
ERROR "GET / HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO "GET /admin HTTP/1.1" 301 0
INFO "GET /admin HTTP/1.1" 301 0
INFO "GET /admin/ HTTP/1.1" 302 0
INFO "GET /admin/ HTTP/1.1" 302 0
INFO "GET /admin/login/?next=/admin/ HTTP/1.1" 200 2194
INFO "GET /admin/login/?next=/admin/ HTTP/1.1" 200 2194
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING Not Found: /static/admin/css/login.css
WARNING Not Found: /static/admin/css/login.css
WARNING "GET /static/admin/css/base.css HTTP/1.1" 404 179
WARNING "GET /static/admin/css/base.css HTTP/1.1" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 404 179
WARNING "GET /static/admin/css/login.css HTTP/1.1" 404 179
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 404 179
WARNING "GET /static/admin/css/login.css HTTP/1.1" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.1" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.1" 404 179
ERROR Internal Server Error: /
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 16, in post_list
    return render(request, 'blog/post_list.html', {'posts': posts})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: blog/post_list.html
ERROR Internal Server Error: /
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 16, in post_list
    return render(request, 'blog/post_list.html', {'posts': posts})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: blog/post_list.html
ERROR "GET / HTTP/1.1" 500 145
ERROR "GET / HTTP/1.1" 500 145
WARNING Not Found: /setor
WARNING Not Found: /setor
WARNING "GET /setor HTTP/1.1" 404 179
WARNING "GET /setor HTTP/1.1" 404 179
WARNING Not Found: /setor/
WARNING Not Found: /setor/
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING "GET /setor/ HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/urls.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /setor/
WARNING Not Found: /setor/
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING Not Found: /setor/
WARNING Not Found: /setor/
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING Not Found: /setor/
WARNING Not Found: /setor/
WARNING "GET /setor/ HTTP/1.1" 404 179
WARNING "GET /setor/ HTTP/1.1" 404 179
INFO "GET /setor/utilidades/ HTTP/1.1" 200 22
INFO "GET /setor/utilidades/ HTTP/1.1" 200 22
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
WARNING Not Found: /setor/utilidades/
WARNING Not Found: /setor/utilidades/
WARNING "GET /setor/utilidades/ HTTP/1.1" 404 179
WARNING "GET /setor/utilidades/ HTTP/1.1" 404 179
WARNING Not Found: /setor/padrao/
WARNING Not Found: /setor/padrao/
WARNING "GET /setor/padrao/ HTTP/1.1" 404 179
WARNING "GET /setor/padrao/ HTTP/1.1" 404 179
INFO "GET /admin/blog/setor/ HTTP/1.0" 200 7422
INFO "GET /admin/blog/setor/ HTTP/1.0" 200 7422
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/blog/setor/1/change/ HTTP/1.0" 200 8224
INFO "GET /admin/blog/setor/1/change/ HTTP/1.0" 200 8224
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "POST /admin/blog/setor/1/change/ HTTP/1.0" 302 0
INFO "POST /admin/blog/setor/1/change/ HTTP/1.0" 302 0
INFO "GET /admin/blog/setor/ HTTP/1.0" 200 7618
INFO "GET /admin/blog/setor/ HTTP/1.0" 200 7618
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug}, product, setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/http/response.py", line 567, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
TypeError: 'QuerySet' object is not callable
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug}, product, setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/http/response.py", line 567, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
TypeError: 'QuerySet' object is not callable
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_json_dict()})
AttributeError: 'Setor' object has no attribute 'to_json_dict'
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_json_dict()})
AttributeError: 'Setor' object has no attribute 'to_json_dict'
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_json_dict()})
AttributeError: 'Setor' object has no attribute 'to_json_dict'
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_json_dict()})
AttributeError: 'Setor' object has no attribute 'to_json_dict'
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO "GET /admin/blog/setor/1/change/ HTTP/1.0" 200 8239
INFO "GET /admin/blog/setor/1/change/ HTTP/1.0" 200 8239
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 18
INFO "GET /setor/padrao/ HTTP/1.1" 200 18
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 18
INFO "GET /setor/padrao/ HTTP/1.1" 200 18
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 91
INFO "GET /setor/padrao/ HTTP/1.1" 200 91
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 66, in setor_detail
    product = Product.objects.filter(Setor=setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 942, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 962, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 969, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1358, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1377, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1258, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1084, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1481, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'Setor' into field. Choices are: id, setor, setor_id, title
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 66, in setor_detail
    product = Product.objects.filter(Setor=setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 942, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 962, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 969, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1358, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1377, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1258, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1084, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1481, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'Setor' into field. Choices are: id, setor, setor_id, title
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 66, in setor_detail
    product = Product.objects.filter(Setor=setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 942, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 962, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 969, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1358, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1377, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1258, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1084, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1481, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'Setor' into field. Choices are: id, setor, setor_id, title
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 66, in setor_detail
    product = Product.objects.filter(Setor=setor)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 942, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 962, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, *args, **kwargs)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/query.py", line 969, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1358, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1377, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1258, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1084, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/db/models/sql/query.py", line 1481, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'Setor' into field. Choices are: id, setor, setor_id, title
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 91
INFO "GET /setor/padrao/ HTTP/1.1" 200 91
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 107
INFO "GET /setor/padrao/ HTTP/1.1" 200 107
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO "GET /admin/blog/product/ HTTP/1.0" 200 6293
INFO "GET /admin/blog/product/ HTTP/1.0" 200 6293
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/blog/product/add/ HTTP/1.0" 200 8048
INFO "GET /admin/blog/product/add/ HTTP/1.0" 200 8048
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "POST /admin/blog/product/add/ HTTP/1.0" 302 0
INFO "POST /admin/blog/product/add/ HTTP/1.0" 302 0
INFO "GET /admin/blog/product/ HTTP/1.0" 200 7664
INFO "GET /admin/blog/product/ HTTP/1.0" 200 7664
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_dict_json(), 'products':[p.to_dict_json() for p in products]})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/http/response.py", line 567, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/serializers/json.py", line 105, in default
    return super().default(o)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Setor is not JSON serializable
ERROR Internal Server Error: /setor/padrao/
Traceback (most recent call last):
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gustavomezzomo/projeto_mercado/blog/views.py", line 67, in setor_detail
    return JsonResponse({'slug':slug, 'setor':setor.to_dict_json(), 'products':[p.to_dict_json() for p in products]})
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/http/response.py", line 567, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/gustavomezzomo/.virtualenvs/git/lib/python3.8/site-packages/django/core/serializers/json.py", line 105, in default
    return super().default(o)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Setor is not JSON serializable
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
ERROR "GET /setor/padrao/ HTTP/1.1" 500 145
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /setor/padrao/ HTTP/1.1" 200 126
INFO "GET /setor/padrao/ HTTP/1.1" 200 126
WARNING Not Found: /favicon.ico
WARNING Not Found: /favicon.ico
WARNING "GET /favicon.ico HTTP/1.1" 404 179
WARNING "GET /favicon.ico HTTP/1.1" 404 179
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO "GET /admin/blog/product/ HTTP/1.0" 200 7442
INFO "GET /admin/blog/product/ HTTP/1.0" 200 7442
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/base.css
WARNING Not Found: /static/admin/css/changelists.css
WARNING Not Found: /static/admin/css/changelists.css
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/base.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/changelists.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/changelists.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING Not Found: /static/admin/css/nav_sidebar.css
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/nav_sidebar.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/js/core.js
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/js/core.js
WARNING Not Found: /static/admin/js/admin/RelatedObjectLookups.js
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/admin/RelatedObjectLookups.js
WARNING "GET /static/admin/js/core.js HTTP/1.0" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING "GET /static/admin/js/core.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.0" 404 179
INFO "GET /admin/jsi18n/ HTTP/1.0" 200 3187
WARNING Not Found: /static/admin/js/urlify.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/jquery.init.js
WARNING Not Found: /static/admin/js/urlify.js
WARNING Not Found: /static/admin/js/jquery.init.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/urlify.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/urlify.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/jquery.init.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/jquery.init.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/jquery/jquery.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING Not Found: /static/admin/js/nav_sidebar.js
WARNING Not Found: /static/admin/js/jquery.init.js
WARNING Not Found: /static/admin/js/jquery.init.js
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/nav_sidebar.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/jquery.init.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/jquery.init.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/core.js
WARNING Not Found: /static/admin/js/core.js
WARNING "GET /static/admin/js/core.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/core.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/admin/RelatedObjectLookups.js
WARNING Not Found: /static/admin/js/admin/RelatedObjectLookups.js
WARNING "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/actions.min.js
WARNING Not Found: /static/admin/js/actions.min.js
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/actions.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/urlify.js
WARNING Not Found: /static/admin/js/urlify.js
WARNING "GET /static/admin/js/urlify.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/urlify.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING Not Found: /static/admin/js/prepopulate.min.js
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/prepopulate.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING Not Found: /static/admin/js/vendor/xregexp/xregexp.min.js
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.0" 404 179
WARNING Not Found: /static/admin/css/responsive.css
WARNING Not Found: /static/admin/css/responsive.css
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
WARNING "GET /static/admin/css/responsive.css HTTP/1.0" 404 179
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/blog/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
INFO /home/gustavomezzomo/projeto_mercado/projeto_mercado/settings.py changed, reloading.
